sdk/core/perf/inc/azure/perf/dynamic_test_options.hpp

```diff
@@ -44,10 +44,7 @@ namespace Azure { namespace Perf {
     {
       try
       {
-        if (m_results[optionName])
-        {
-          return m_results[optionName].as<T>();
-        }
+        return m_results[optionName].as<T>();
```

Comment:
I don't see how the program can get to line 57 anymore, which is fine, but maybe some tools would start complaining about dead code.
Do you think it makes sense for `catch (argagg::unknown_option const&)` to do `{}` instea of `{ return defaultValue; }`? This way, control will get to line 57, which does the same thing.